sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/msfaa-integration/msfaa-full-time-process-response-integration.scheduler.ts

```diff
@@ -0,0 +1,48 @@
+import { InjectQueue, Process, Processor } from "@nestjs/bull";
+import { MSFAAResponseService } from "@sims/integrations/esdc-integration";
+import { QueueService } from "@sims/services/queue";
+import { OfferingIntensity } from "@sims/sims-db";
+import { QueueNames } from "@sims/utilities";
+import { Job, Queue } from "bull";
+import { QueueProcessSummary } from "../../../models/processors.models";
+import { BaseScheduler } from "../../base-scheduler";
+import { ProcessResponseQueue } from "../models/esdc.dto";
+
+@Processor(QueueNames.FullTimeMSFAAProcessResponseIntegration)
+export class FullTimeMSFAAProcessResponseIntegrationScheduler extends BaseScheduler<void> {
+  constructor(
+    @InjectQueue(QueueNames.FullTimeMSFAAProcessResponseIntegration)
+    schedulerQueue: Queue<void>,
+    queueService: QueueService,
+    private readonly msfaaResponseService: MSFAAResponseService,
+  ) {
+    super(schedulerQueue, queueService);
+  }
+
+  /**
+   * Download all fulltime files from MSFAA Response folder on SFTP and process them all.
+   * @params job job details.
+   * @returns Summary with what was processed and the list of all errors, if any.
+   */
+  @Process()
+  async processMSFAA(job: Job<void>): Promise<ProcessResponseQueue[]> {
+    const summary = new QueueProcessSummary({
+      appLogger: this.logger,
+      jobLogger: job,
+    });
+    await summary.info(
```

Comment:
Ideally, all the schedulers should have a log before and also after the process is completed. I saw some following this and some not following.